### PR TITLE
Updated NAT Loopback issues section with explanations 

### PR DESCRIPTION
**Pull request by jzold**

### CODE REVIEW

**Comment by xlionjuan** [comment]
First, terms

* Rustdesk -> RustDesk 
* Nat -> NAT

For the con

**Comment by xlionjuan** [comment]
First, terms

* Rustdesk -> RustDesk 
* Nat -> NAT

For the configuration for the router's NAT Loopback, is this kind of options available on most home router? As far as I know, some of them are preconfigured and enabled NAT Loopback, some of them are disabled, and both of them don't have the options to control this functionality.

And I'm more preferred `example.com`, because it is reserved domain, and can using it without security concerns.

**Comment by jzold** [comment]
@xlionjuan  good spot on the "Nat" vs "NAT" inconsistency, I'm amending the PR with the fix  soon. Same for "Rustdesk". While we're at terms: 
- consistency: I've spotted a few cases in this repo with "Rustdesk" instead of "RustDesk" (mainly in other language translations) that I recommend amending ;)

I'm not clear on your last point: the PR contains an example FQDN ("myowndomain.com") as an illustrative example only. What's your security concern?

**Comment by xlionjuan** [comment]
> What's your security concern?

> because it is reserved domain

**Comment by jzold** [comment]
myowndomain.com is a registered domain and in the context of a RustDesk documentation using this domain as an example I see no security concerns. We could consider this case as another consistency principle of using example.com across the documentation when referring to external FQDNs? :D

**Comment by jzold** [comment]
need some cosmetic fixes

**Comment by xlionjuan** [comment]
> For the configuration for the router's NAT Loopback, is this kind of options available on most home router? As far as I know, some of them are preconfigured and enabled NAT Loopback, some of them are disabled, and both of them don't have the options to control this functionality.

This?

**Comment by jzold** [comment]
@xlionjuan  NAT loopback is usually a feature rather than a config for SOHO routers they either support it or don't. On that basis I recommend amending the RustDesk documentation further because the "Set up NAT Loopback on your router" section in the  doc  kind of indicates that NAT loopback is a configuration rather than a feature ;). 

There are cases where NAT loopback should be configured (eg. unify/ubiquity https://help.ui.com/hc/en-us/articles/204952134-EdgeRouter-Hairpin-NAT#2) - this is not a typical SOHO router though :) 

The main point is: NAT loopback is a feature but it needs configuration to make it work (eg. SNAT/port forwarding etc.) and on that basis I think the documentation should emphasise "additional configuration" as the main point. And the doc should also emphasise to avoid NAT loopback with workarounds (which is why I added "workarounds" in the "Solutions" section in my PR).

Any thoughts?

**Comment by xlionjuan** [comment]
I think if users that using any business or semi-business grade router, they should know *(or know where to find)* the solution, we should not guide user *"How to enable NAT Loopback"*.

tl;dr explain what NAT Loopback is acceptable, but shouldn't detail, to prevent the potential to take the responsibility for misconception or misleading.
